src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -90,7 +95,42 @@ class SurrealPersistenceService implements PersistenceService {
         // create wave_scan table
         final ret4 = surrealDb.sqlAsMap(authorization, "define table wave_scan_vuln SCHEMALESS")
         if( ret4.status != "OK")
-            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret3")
+            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret4")
+
+        try {
+            // create wave_matrics_pull  table
+            surrealDb.sqlAsString(authorization, '''
+                                                    DEFINE TABLE wave_metrics_pull SCHEMAFULL;
+                                                    DEFINE FIELD count ON TABLE wave_metrics_pull TYPE number;
+                                                    DEFINE FIELD ip ON TABLE wave_metrics_pull TYPE string; 
+                                                    DEFINE FIELD userid ON TABLE wave_metrics_pull TYPE number;
+                                                    DEFINE FIELD imagename ON TABLE wave_metrics_pull TYPE string;
+                                                    DEFINE FIELD date ON TABLE wave_metrics_pull TYPE datetime; 
```

Comment:
Securely this is faster, but the those info could not be extracted using an aggregation of `wave_request` and `wave_build` tables?

Reply:
I will look into it

Reply:
I can get the metrics without dates from `wave_build`, but it's not working when supplying dates.
Because I think currently surreal is treating timestamp in wave build records as String 
I will dig further and see if there is a solution for this

Reply:
This sound a bit weird. I've made some queries in the past over a time range. If I'm not wrong it should just work

Reply:
I made this in the past 

```
select count() as total, time::month(type::datetime(timestamp)) as month from wave_request group by month limit 10
```

Reply:
it indeed solves the date issue, i will use the current tables and wont create new ones